clang/lib/CIR/CodeGen/CIRGenStmtOpenACC.cpp

```diff
@@ -304,12 +304,21 @@ CIRGenFunction::emitOpenACCCacheConstruct(const OpenACCCacheConstruct &s) {
   return mlir::success();
 }
 
+const VarDecl *getLValueDecl(const Expr *E) {
+  // We are going to assume that after stripping implicit casts, that the LValue
+  // is just a DRE around the var-decl.
+
+  E = E->IgnoreImpCasts();
+
+  const auto *DRE = cast<DeclRefExpr>(E);
```

Comment:
```suggestion
  const auto *dre = cast<DeclRefExpr>(e);
```
I'm sure you must love flipping between two coding styles in the same PR. 😄

Reply:
As you can tell, it gets me all the time :D